string name
---
server_slam/Pose pose
server_slam/PaArray2D msg2d
